src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -296,11 +301,15 @@
       <Link>Common\Interop\OSX\Interop.Libraries.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsFreeBSD)' == 'true' OR '$(TargetsUnknownUnix)' == 'true' ">
+  <ItemGroup Condition=" ('$(TargetsFreeBSD)' == 'true' OR '$(TargetsUnknownUnix)' == 'true') AND '$(IsPartialFacadeAssembly)'!='true'">
```

Comment:
TargetGroup net46/net461 doesn't imply TargetsWindows. The OSGroup=Windows_NT implies TargetsWindows. These facade project are currently targeting AnyOS which is why it is needed here.